src/lib/datepicker/datepicker.spec.ts

```diff
@@ -96,6 +95,20 @@ describe('MdDatepicker', () => {
         expect(document.querySelector('md-dialog-container')).not.toBeNull();
       }));
 
+      it('open in disabled mode should not open the calendar', async(() => {
```

Comment:
At one point I think calling `open` required it to be async, I'm not sure if that's still the case